src/Dapper.AOT/TypeAccessor.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Dapper.Internal;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace Dapper;
+
+public static class TypeAccessor
```

Comment:
@mgravell can you, please, confirm this is the proper API we want to expose and we want exactly the `TypeAccessor<T>` api implementation to be generated per type?

Reply:
can be added in next PR